extensions/default/src/ViewerLayout/Header.jsx

```diff
@@ -42,16 +60,29 @@ function Header({ children }) {
           <span className="mr-3 text-lg text-common-light">
             FOR INVESTIGATIONAL USE ONLY
           </span>
-          <IconButton
-            variant="text"
-            color="inherit"
-            className="text-primary-active"
-            onClick={() => {}}
-          >
-            <React.Fragment>
-              <Icon name="settings" /> <Icon name="chevron-down" />
-            </React.Fragment>
-          </IconButton>
+          <Dropdown
+            titleElement={
+              <IconButton
+                variant="text"
+                color="inherit"
+                size="initial"
+                className="text-primary-active"
+                onClick={() => {}}
+              >
+                <React.Fragment>
+                  <Icon name="settings" /> <Icon name="chevron-down" />
+                </React.Fragment>
+              </IconButton>
+            }
+            list={[
+              { title: 'About', icon: 'info', onClick: () => showAboutModal() },
```

Comment:
Tip of the day 😄  i think you know about it: We should avoid passing arrow funcs directly to UI elements whenever possible, I know this is just an initial step towards final implementation but i think we can train ourselves to define them in the nearest component or parent as a handler and use the handler instead. This help us avoid rerenderings since the arrow func will be attached to the component instance.

Reply:
Are you mentioning to have methods like `showAboutModal` centralized in the top-parent component and then pass to the `Header` as props?

https://github.com/OHIF/Viewers/blob/140c562519ed9173d53d3fea06cb08ca867b1ea3/extensions/default/src/ViewerLayout/index.jsx#L82

Reply:
Are you mentioning to have methods like `showAboutModal` centralized in the top-parent component and then pass to the Header as props? Here for example:

https://github.com/OHIF/Viewers/blob/140c562519ed9173d53d3fea06cb08ca867b1ea3/extensions/default/src/ViewerLayout/index.jsx#L82

We could even maybe think in a better way to split/reuse this Header component since it's being used in `extensions/default` and `platform/viewer`. I'll take a look at it tomorrow :D

Reply:
@rodrigolabs I mean talking about js itself and React, arrow functions is basically a function declaration so every render this will be a different point in memory that could possibly cause unnecessary renders. Moving the declaration to the nearest parent or any component so we have a function instance for each component instance, this way we are not redeclaring functions in each rerender.

Reply:
oh gotcha, you're right! for this scenario, we can use `useCallback`.